job/dhcp.go

```diff
@@ -28,6 +28,13 @@ func IsSpecialOS(i *packet.Instance) bool {
 // ServeDHCP responds to DHCP packets
 func (j Job) ServeDHCP(w dhcp4.ReplyWriter, req *dhcp4.Packet) bool {
 
+	// If we are not the chosen provisioner for this piece of hardware
+	// do not respond to the DHCP request
+	if !j.areWeProvisioner() {
+		j.Error(errors.New("not the provisioner for this piece of hardware"))
```

Comment:
Should be `j.With("error", errors.New(...)).Info()`

Reply:
I'm not sure we really care to log this though? This is going to be spammy right?

Reply:
Yeah I figured we might end up not logging this at all as it will be majorly spammy...   but we already are majorly spammy, so I thought maybe... ;).     I'll just ditch the logging.